simpeg/inverse_problem.py

```diff
@@ -64,6 +64,7 @@ def beta(self):
     @beta.setter
     def beta(self, value):
         self._beta = validate_float("beta", value, min_val=0.0)
+        self.reg.multiplier = self._beta
```

Comment:
This could let us treat the functions in the global objective function as just a flat list of functions (`mult1 * misfit1 + mult2 * misfit2 + beta * multi_a * reg_a + beta * multi_b * reg_b + ...` )